api/permissions/rbac_wrapper.py

```diff
@@ -23,3 +30,57 @@ def get_roles_permission_data_for_environment(*args, **kwargs):
 
     def get_role_permission_filter(*args, **kwargs) -> Q:
         return Q()
+
+
+def is_master_api_key_object_admin(
+    master_api_key: "MasterAPIKey", object_: Union[Project, Environment]
+) -> bool:
+    if not settings.IS_RBAC_INSTALLED:
+        return False
```

Comment:
Maybe I missed this somewhere already, but they can still be an admin via the org level admin role, right?

Reply:
Roles don't support organisation admin, a key can be organisation admin directly though and that is handled in permission_service

Reply:
Yep, thought so 👍